src/EVM/Expr.hs

```diff
@@ -1307,9 +1314,11 @@ simplifyProp prop =
     go (PNeg (PGEq a b)) = PLT a b
     go (PNeg (PLT a b)) = PGEq a b
     go (PNeg (PLEq a b)) = PGT a b
+    go (PNeg (PAnd a b)) = POr (PNeg a) (PNeg b)
```

Comment:
If we are adding this, shouldn't we add the dual `go (PNeg (POr a b)) = PAnd (PNeg a) (PNeg b)` as well?

Reply:
Yes! Thanks, yeah, I just added this, along with a test case :)